dll/cpl/sysdm/environment.c

```diff
@@ -4,10 +4,17 @@
  * FILE:        dll/cpl/sysdm/environment.c
  * PURPOSE:     Environment variable settings
  * COPYRIGHT:   Copyright Eric Kohl
+ *              Copyright Arnav Bhatt <arnavbhatt288@gmail.com>
  *
  */
 
 #include "precomp.h"
+#include <commdlg.h>
+#include "commctrl.h"
+#include <string.h>
+#include <strsafe.h>
+
+#define CX_LIST 30160  // Make columns ridiculously long in order to avoid resizing it
```

Comment:
looks like a hack

Reply:
@HBelusca The reason why is this because since whenever the listview control gets resized (when in this case, the dialog box itself gets resized) it is not ideal to resize the rows of listview controls as well since it doesn't get resized properly. Hence making rows big earlier on is a better choice.

Reply:
@HBelusca I figured out the problem and removed the hack. You may now resolve this review.